codes/csharp/chapter_searching/binary_search_edge.cs

```diff
@@ -20,7 +20,7 @@ public class binary_search_edge {
     }
 
     /* 二分查找最右一个 target */
-    public int BinarySearchRightEdge(int[] nums, int target) {
+    public static int BinarySearchRightEdge(int[] nums, int target) {
```

Comment:
We use static function in Java because we have to. Is is necessary for C#?

Reply:
How about keep all static usage same with Java code?

Reply:
I agree. Both adding or not adding 'static' are acceptable for me. Generally, I'd like to keep the code as clean(in other words, short) as possible.

Reply:
fixed